lib/model/book.dart

```diff
@@ -63,7 +66,13 @@ Book toBook(DocumentSnapshot doc) {
   id: doc.id,
   title: data['title'],
   cover: data['cover'],
+  state: BookState.values[data['state'] ?? 0],
   createdAt: DateTime.fromMillisecondsSinceEpoch(data['createdAt'] ?? 0),
   modifiedAt: DateTime.fromMillisecondsSinceEpoch(data['modifiedAt'] ?? 0),
   );  else return null;
 }
+
+enum BookState {
```

Comment:
As you can see to handle the book delete, I just added a state as with the notes. I think we can have trash item on profile screen to let the user see the deleted books or notes

Reply:
Yeah that's a pretty good idea

Reply:
Let's envision it for our second iteration.